docs/concepts.rst

```diff
@@ -122,7 +111,7 @@ Many of the most common flows you'll work with in CumulusCI are designed to buil
 * ``dev_org``: This is an unmanaged org that is designed for development use. This flow is typically used with an org whose configuration is ``dev`` or ``dev_namespaced``.
 * ``qa_org``: This is an unmanaged org that is designed for testing. This flow is typically used with an org whose configuration is ``qa``.
 * ``install_beta``: This is a managed org with the latest beta release installed, for projects that build managed packages. This flow is typically used with an org whose configuration is ``beta``.
-* ``install_prod``: This is a managed org with the latest release installed, for projects that build managed packages. 
-* ``regression_org``: This is a managed org that starts with the latest release installed and is then upgraded to the latest beta to simulate a subscriber upgrade, for projects that build managed packages. This flow is typically used with an org whose configuration is ``release``.
+* ``install_prod``: This is a managed org with the latest release installed, for projects that build managed packages.
+* ``regression_org``: This is a managed org that starts with the latest release installed and is then upgraded to the latest beta to simulate a subscriber upgrade for projects that build managed packages. This flow is typically used with an org whose configuration is ``release``.
 
-CumulusCI derives the library of flows and tasks available for any project by combining its internal, out-of-the-box library with the customizations in ``cumulusci.yml``. Customizations can add new tasks and flows, customize the way out-of-the-box tasks behave, and extend, combine, and modify out-of-the-box flows to suit the specific needs of the project. We cover customization in depth in TODO: reference "Customizing CumulusCI"
+CumulusCI derives the library of flows and tasks available for any project by combining its internal standard library with your customizations in ``cumulusci.yml``. Customizations can add new tasks and flows, customize the way featured tasks behave, and extend, combine, and modify featured flows to suit the specific needs of the project. We cover customization in depth in [TODO: reference Customizing CumulusCI].
```

Comment:
I'm not understanding your use of "featured" here.

Reply:
@davisagli I don't like _out-of-the-box_, thought _featured_ would be a better substitute. I can change it back, or is there a better word?

Reply:
out-of-the-box and featured don't seem like synonyms to me. The point we're trying to make is that a project can either use the standard tasks and flows as is, or customize them, or add new ones. Any of the standard tasks and flows can be customized, not just "featured" ones.

Reply:
Then we won't use either, simply go with tasks.